src/util_i_varlists.nss

```diff
@@ -1066,6 +1108,114 @@ void SetListString(object oTarget, int nIndex, string sValue, string sListName =
         SetLocalString(oTarget, LIST_REF + sListName + IntToString(nIndex), sValue);
 }
 
+int CopyListFloat(object oSource, object oTarget, string sSourceName, string sTargetName, int nIndex, int bAddUnique = FALSE, int nRange = 1)

```

Comment:
I would consider swapping the order of the nRange and bAddUnique parameters to keep like with like.

Reply:
Fixed in all functions and prototypes.